corehq/apps/cloudcare/templates/cloudcare/partials/form_entry/bootstrap5/entry_signature.html

```diff
@@ -14,7 +15,7 @@
   <button class="btn btn-outline-primary btn-sm float-end" data-bind="click: onClear">
     {% trans "Clear" %}
   </button>
-  <input type="file" class="hidden" data-bind="
+  <input type="file" class="d-none" data-bind="
```

Comment:
This is a little different than other things being done in [this commit](https://github.com/dimagi/commcare-hq/pull/34631/commits/e09df57ca7800c280f70cabf3a9cb2341f4154e7?diff=unified&w=1). I assume it fixes an oversight that was easy to address here, is that correct?

Aside (opinion): not sure who's responsible for it, but how is `d-none` better than `hidden`? This seems like gratuitous name changing.